prescription.sh

```diff
@@ -96,6 +96,10 @@ function generateYML () {
         --coverity.stream=*) coverity_stream="${i#*=}" ;;
         --coverity.username=*) coverity_username="${i#*=}" ;;
         --coverity.password=*) coverity_password="${i#*=}" ;;
+        --codedx.url=*) codedx_server_url="${i#*=}" ;;				#codedx
```

Comment:
In manifest file the codedx url is referenced as `<<CODEDX_INSTANCE_URL>>` Can we change `codedx_server_url` to `codedx_instance_url` to keep the variables name consistent with the name in manifest file ?

Reply:
Changed `CODEDX_INSTANCE_URL` to `CODEDX_SERVER_URL` now to keep it consistent with other tools url related placeholders

Reply:
Thanks